tests/components/vizio/test_config_flow.py

```diff
@@ -309,3 +355,33 @@ def vizio_cant_connect_fixture():
 
     assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
     assert result["reason"] == "already_setup"
+
+
+async def test_import_flow_update_options(
+    hass: HomeAssistantType, vizio_connect, vizio_bypass_update
+) -> None:
+    """Test import config flow with updated options."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": SOURCE_IMPORT},
+        data=vol.Schema(VIZIO_SCHEMA)(MOCK_IMPORT_VALID_TV_CONFIG),
+    )
+    await hass.async_block_till_done()
+    assert result["result"].options == {CONF_VOLUME_STEP: VOLUME_STEP}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4847616)